sims/common-sim-flags.mk

```diff
@@ -3,6 +3,17 @@
 #----------------------------------------------------------------------------------------
 SIM_OPT_CXXFLAGS := -O3
 
+# Workaround: esp-isa-sim doesn't install libriscv,
```

Comment:
What is `libriscv` used for again? What would break?

Reply:
Its a dynamic library installed by spike. Nothing should break.

Reply:
What I mean in the comment is that building `SpikeTileConfig` after the user has installed `esp-isa-sim`, but not `riscv-isa-sim`, will cause a link failure.